### PR TITLE
React  capstone project

### PR DESCRIPTION
**Pull request by mandelatuks**
### In this capstone, I did the following:

-  First I to choose an API [covid19 api](https://disease.sh/v3/covid-19/countries) to base the development of the web app on
- create react app
- install redux and implement API calls
- design the layout to match the template's structure.
- show covid19 data on the home page
- Create a Details page and render data according to what the user has selected
- Update Readme and documentation
- Add video documentation
- no linters
- Loom [video](https://www.loom.com/share/7c9ed7b919cc400cbf21ba073ad5e462)

### CODE REVIEW

File: MIT.md [hunk 1]
```diff
@@ -1,10 +0,0 @@
-## Copyright 2021, [YOUR NAME]
```
Comment (rustam-tolipov) [inline MIT.md:1]: - [x] [OPTIONAL] Please kindly fill your MIT license file including your name and change `[APP TYPE]` to web app. If I understood well  it should be in your dev branch

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap');
+
+body {
+  margin: 0;
+}
```
Comment (rustam-tolipov) [inline src/App.css:5]: - [ ] [OPTIONAL] I see you imported the Lato font. Please to make it work add `font-family: 'Lato', sans-serif;` to the body tag in CSS

**Review by rustam-tolipov — APPROVED** [review]
## Hi @mandelatuks 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.css (hunk 2 above)
Comment (mandelatuks) [reply src/App.css:5]: Thank you working on it now

**Pull request merged**